test/apis/placement/v1/resource_validation_test.go

```diff
@@ -0,0 +1,219 @@
+/*
+Copyright (c) Microsoft Corporation.
+Licensed under the MIT license.
+*/
+package v1
+
+import (
+	"errors"
+	"fmt"
+	"reflect"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	rbacv1 "k8s.io/api/rbac/v1"
+	k8serrors "k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	clusterv1 "go.goms.io/fleet/apis/cluster/v1"
+)
+
+var _ = Describe("Resource validation tests for denying Member Cluster", func() {
```

Comment:
please refer to the test/apis/placement/v1alpha1/api_validation_integration_test.go.

The structure is
```
describe,
    context
         it
         it
         it
    context
         it
         it
```